doc/release/release_dev.rst

```diff
@@ -208,53 +227,94 @@ A total of 217 changes have been committed.
 - PR for issue #6033 Improve test coverage for algorithms in betweenness_subset.py #6033 (#6083)
 - Di graph edges doc fix (#6108)
 - Improve coverage for core.py (#6116)
-- Add clear_edges() method to the list of methods to be frozen by nx.freeze() (#6190)
+- Add clear edges method to the list of methods to be frozen by the nx.… (#6190)
```

Comment:
The formatting of this one looks a little weird with the unicode ellipsis, but no big deal.

Reply:
I changed the title of the #6190 PR to be `Add clear edges method as a method to be frozen by nx.freeze`.  So:
```suggestion
- Add clear edges method as a method to be frozen by nx.freeze (#6190)
```
This should be what the script produces now.